models/api_host.go

```diff
@@ -94,6 +97,10 @@ func (a *ApiHost) ConvertAPIHostToNMHost(currentHost *Host) *Host {
 	h.IsDefault = a.IsDefault
 	h.NatType = currentHost.NatType
 	h.TurnEndpoint = currentHost.TurnEndpoint
-
+	if h.PersistentKeepalive == 0 {
+		h.PersistentKeepalive = time.Duration(a.PersistentKeepalive)
+	} else {
+		h.PersistentKeepalive = currentHost.PersistentKeepalive
+	}
```

Comment:
h.PersistentKeepalive = a.PersistentKeepalive

Reply:
https://github.com/gravitl/netmaker/pull/2604/commits/c02eb3a8a7cc7b702bfc9959b2b5cbc80e2ad99d

Reply:
the above func is called for host update from UI so, there is no need to check h.PersistentKeepalive == 0, just set h.PersistentKeepalive = a.PersistentKeepalive

Reply:
https://github.com/gravitl/netmaker/pull/2604/commits/a51ddebde7885c4150be31c33bb484cca518efa2